.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Latest Documentation
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
```

Comment:
Thanks! Can you actually add it to this file instead (https://github.com/rjsf-team/react-jsonschema-form/blob/master/.github/workflows/release.yml)? That way we just have one file for release CI tasks, would be a bit simpler.